### PR TITLE
Created folder structure and added Navbar

### PR DESCRIPTION
**Pull request by nargis764**
Created folder structure and added Navbar, removed tailwind package

### CODE REVIEW

File: client/postcss.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (ProwlingLynx) [inline client/postcss.config.js:1]: We need to remove this file if tailwind is not being used.

File: client/src/App.js [hunk 2]
```diff
@@ -6,19 +6,32 @@ import {
...
+import Navbar from "./components/Navbar"
 
-const App = () => {
-  const routes = useRoutes([
-    { path: "/", element: <Home /> },
-    // ...
-  ]);
-  return routes;
-};
+// const App = () => {
```
Comment (ProwlingLynx) [inline client/src/App.js:14]: If we don't need this code, we should remove it.

File: client/src/components/Navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { Link } from "react-router-dom";
+import styles from "./Navbar.module.css";
+
+
+const Navbar = () => {
+
+
+    return (
+        <div>
+            {/* <ul className="flex flex-row justify-between md:px-[10%] lg:px-[20%] text-center mt-[5%]"> */}
```
Comment (ProwlingLynx) [inline client/src/components/Navbar/Navbar.js:11]: Clean up commented code plz.

File: client/src/components/Navbar/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            <ul className={styles.navbar}>
+                {/* <li className="border-r-2 border-black px-10">Home</li>
```
Comment (ProwlingLynx) [inline client/src/components/Navbar/Navbar.js:13]: Comment removal from the whole file .

File: client/src/components/Navbar/Navbar.module.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    align-items:center;
+    list-style-type: none;   
+    padding:0 20%;
+}
+
+.navlink {
+    text-decoration:none;
+}
+
+@media only screen and (min-width: 800px) and (max-width: 1200px) {
```
Comment (ProwlingLynx) [inline client/src/components/Navbar/Navbar.module.css:14]: Nice Work on the media query.

Question: are you constraining this query for a tablet size 800 -1200 wide?

File: client/src/components/Wrapper/Wrapper.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import styles from "./Wrapper.module.css";
+
+
+const Wrapper = ({ children, style = {} }) => {
```
Comment (ProwlingLynx) [inline client/src/components/Wrapper/Wrapper.js:5]: What problem are you trying to solve by using this component?

File: client/src/components/Wrapper/Wrapper.module.css [hunk 7]
```diff
@@ -0,0 +1,29 @@
+.wrapper {
```
Comment (ProwlingLynx) [inline client/src/components/Wrapper/Wrapper.module.css:1]: Same comment as above in the component code.

File: client/package.json [hunk 8]
```diff
@@ -35,5 +35,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "autoprefixer": "^10.4.8",
```
Comment (ProwlingLynx) [inline client/package.json:40]: Do we still need these dependencies?

File: client/src/pages/Home/Home.js [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+        id:4,
+        name:"BBQ Wings",
+        posted_at:"9/9/2009"
+    }
+]
+
+const Home = () => {
+    return (
+        <div className="">
+            <div className="flex flex-row justify-between">
```
Comment (ProwlingLynx) [inline client/src/pages/Home/Home.js:33]: The class name makes me think this is a remainder from tailwind.

**Review by ProwlingLynx — CHANGES_REQUESTED** [review]
Good work overall. I recommend removing the comments first and also a second look at the wrapper component. I've marked most instances with comments.

**Comment by ProwlingLynx** [comment]
Forgive the double post here. I also recommend practicing making more commits. It will help give a better idea of what was done to better track feature development and tickets.

File: client/postcss.config.js (hunk 1 above)
Comment (nargis764) [reply client/postcss.config.js:1]: Good Point, I am going to remove it soon.

File: client/src/components/Navbar/Navbar.js (hunk 3 above)
Comment (nargis764) [reply client/src/components/Navbar/Navbar.js:11]: Sure!

File: client/src/components/Navbar/Navbar.module.css (hunk 5 above)
Comment (nargis764) [reply client/src/components/Navbar/Navbar.module.css:14]: For Now, I just put it like that and was playing around, but it is subject to change later as we start working on responsiveness

File: client/src/components/Wrapper/Wrapper.js (hunk 6 above)
Comment (nargis764) [reply client/src/components/Wrapper/Wrapper.js:5]: We will discuss that in the group chat elaborately

**Pull request closed without merging**